units/provisioning_create_host.go

```diff
@@ -337,7 +337,6 @@ func (j *createHostJob) createHost(ctx context.Context) error {
 		}
 	}
 
-	event.LogHostStartFinished(j.host.Id, true)
```

Comment:
I've found it helpful to see in the host logs when a task host was started.
would it work to remove logging from the spawnhost_start job instead?

Reply:
Ah I see. I don't want to do it in the other job because we do want to know when a spawnhost has started if it was stopped previously. I'll make a different change though; I was torn originally between which was better.